File: Pertemuan 5/BelajarOperator.java
import java.util.Scanner;
public class BelajarOperator {
    public static void main(String[] args) {
        
        Scanner scanner = new Scanner(System.in);
        System.out.println("Masukkan nilai A = ");
        double a = scanner.nextDouble();
        System.out.println("Masukkan nilai B =  ");
        double b = scanner.nextDouble();

        System.out.println("A == B = " + (a==b));
        System.out.println("A != B = " + (a!=b));
        System.out.println("A > B = " + (a>b));
        System.out.println("A < B = " + (a<b));
        System.out.println("A >= B = " + (a>=b));
        System.out.println("A <= B = " + (a<=b));

        boolean hasil = (a > 5 && b > 2);
        System.out.println("hasil A < 5 && B > 2 = " + hasil);
        System.out.println("hasil A < 5 || B > 2 = " + hasil);
        System.out.println("hasil !(A < 5 && B > 2)" + hasil);
        scanner.close();
    }
}

File: Pertemuan 5/BelajarOperator.py
a = input("Masukkan nilai A = ")
b = input("Masukan nilai B =")

print ("A == B " + str(a==b))
print ("A != B " + str(a!=b))
print ("A <= B " + str(a<=b))
print ("A >= B " + str(a>=b))
print ("A < B " + str(a<b))
print ("A > B " + str(a>b))

x = int(a) < 5 and int(b) > 4
print("A < 5 dan B > 4 = " + str(x))
print("A < 5 dan B > 4 = " + str(int (a) < 5 or int(b) > 4))
print("A < 5 dan B > 4 = " + str(not(int(a) < 5)))
